### PR TITLE
Add metadata to the song

### PR DESCRIPTION
**Pull request by storymode7**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - metadata-manager
+
+install:
+    - pip install pipenv
+    - pipenv install .
+    - pip install flake8
+
+script:
+    - python3 -m  unittest discover tests
+    - flake8 --ignore=F401
```
Comment (CuriousLearner) [inline .travis.yml]: Why to ignore this?

File: CHANGELOG [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[unreleased]
+Added
+- Album cover and icon setter
+- More file name support. Artist can now appear on either side of a -
+- Seperate function to get the song name and artist from
+  various combinations and confirm from spotify database
+- Song without artist name support
+
+[0.1.0] - 2017-03-20
```
Comment (CuriousLearner) [inline CHANGELOG]: Date should be 2018, I guess.

File: .travis.yml (hunk 1 above)
Comment (CuriousLearner) [inline .travis.yml]: Why to ignore this?

File: attachmeta.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
+import findmeta
+import eyed3
+import requests
+
+
+def splitByFirstOccurence(string, character):
```
Comment (CuriousLearner) [inline attachmeta.py]: Name your functions in snake case. This is not Java or JavaScript. Follow PEP8 :)

File: authorize_format [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Save as authorize.py
+import spotipy
```
Comment (CuriousLearner) [inline authorize_format]: Move this to settings, and then pick the value from env variables.

You may use `.env`

File: findmeta.py [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+    while True:
+        choice = int(input("Menu:\n"
+                           "1) Search for a track\n"
+                           "2) See available music categories\n"
+                           "3) See available playlists in a category\n"
+                           "4) Get playlist tracks\n"
+                           "5) Get Track Info\n"
+                           "6) Exit\n"
+                           ))
+        print()
```
Comment (CuriousLearner) [inline findmeta.py]: Remove this

File: authorize_format (hunk 4 above)
Comment (storymode7) [reply authorize_format]: Wow. Just learning how a thing can be done in so many different ways!
And this way is more easier given that pipenv loads .env automatically :)
Fixed in: https://github.com/fossdelhi/sync-music/commit/380bd458ec7a1f2c3a15d461f5cc9de32109085c

File: attachmeta.py (hunk 3 above)
Comment (storymode7) [reply attachmeta.py]: Alright.
Here we go: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650

File: CHANGELOG (hunk 2 above)
Comment (storymode7) [reply CHANGELOG]: Oops. Fixed along with cases: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650

File: .travis.yml (hunk 1 above)
Comment (storymode7) [reply .travis.yml]: Was solved by fixing directory structure. And moving `attachmeta.py` from `src` to project root.
https://github.com/fossdelhi/sync-music/commit/d070f849ab383ced32b0e176237cee8d33a3a8fa

https://github.com/fossdelhi/sync-music/commit
/67ca86c217a0e5ce48745119939b1916efd83a18

File: findmeta.py (hunk 5 above)
Comment (storymode7) [reply findmeta.py]: I now see, there was no use of this except in the initial debugging phases.
Fixed in: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650

**Pull request merged**